### PR TITLE
Some things may interest

### PR DESCRIPTION
**Pull request by pedroalbanese**
- internal stopwatch (we don't have time command on Windows)
- 13 hash digests including GOST94/Streebog for you to choose from

### CODE REVIEW

**Comment by vns0** [comment]
Hi, thanks for the input! It is necessary to resolve conflicts, in addition to conflicts - you do not need to format all the code, either only its part, or according to the already set rules

File: dehash.go [hunk 1]
```diff
@@ -1,176 +1,274 @@
 package main
 
 import (
-    "fmt"
```
Comment (vns0) [inline dehash.go:4]: Please return the formatting so as not to overwrite the past developer

Comment (vns0) [reply dehash.go:4]: You also need to go through the whole project + resolve conflicts

File: go.mod [hunk 2]
```diff
@@ -0,0 +1,16 @@
+module github.com/pedroalbanese/hash-cracker
```
Comment (nikitavoryet) [inline go.mod:1]: It is better not to store such files in the repository, can you delete them?

File: go.sum [hunk 3]
```diff
@@ -0,0 +1,143 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```
Comment (vns0) [inline go.sum:1]: This is a temporary file and should be deleted.

File: main.go [hunk 4]
```diff
@@ -1,56 +1,61 @@
 package main
 
-
 import (
```
Comment (vns0) [inline main.go:3]: Same formatting problem that will overwrite history

**Review by vns0 — COMMENTED** [review]
Please go through the comments, then I will gladly accept the changes =)

File: README.md [hunk 5]
```diff
@@ -1,7 +1,31 @@
...
+    | SHA2            |     |     |     | O   | O   |Fast  | 
+    | SHA3            |     |     |     | O   | O   |Fast  |
+    | Skein512        |     |     |     | O   | O   |Medium|
+    | SM3             |     |     |     | O   |     |Fast  |
+    | Streebog        |     |     |     | O   | O   |Slow  | 
+    | Tiger           |     |     | O   |     |     |Medium| 
+    | Whirlpool       |     |     |     |     | O   |Slow  |
+    
 ## Usage
     ./dehash -hash hexstring|-file path [-type sha1|sha256|sha512|md5] [-max val] [-min val] [-charset chars] [-threads num]
```
Comment (vns0) [inline README.md:30]: All supported algorithms should be added here in the example.

File: README.md [hunk 6]
```diff
@@ -43,22 +67,6 @@ Crack **sha1** hashes from file
     
     ./dehash -type sha1 -file ./hashes.txt
 
-## Get elapsed time
```
Comment (vns0) [inline README.md:46]: The command should be left, it works fine, if there are compatibility problems - write, I will try to fix it!

**Review by vns0 — COMMENTED** [review]
Readme.MD CR

File: README.md (hunk 6 above)
Comment (pedroalbanese) [reply README.md:46]: There is no time command on Windows, that would be a workaround..

**Comment by pedroalbanese** [comment]
Check now please. I left just the faster algorithms (some Go implementations are quite slow)

**Pull request closed without merging**